client/blocks/app-promo/index.jsx

```diff
@@ -185,6 +197,14 @@ export class AppPromo extends Component {
 						) }
 					</p>
 				</Dialog>
+				<Button
+					tabIndex="0"
+					className="app-promo__dismiss"
+					onClick={ this.dismiss }
+					aria-label={ translate( 'Dismiss' ) }
+				>
+					<Gridicon icon="cross" size={ 16 } />
+				</Button>
```

Comment:
This looks repeated from above. Should we abstract it to a function or constant so we don't repeat it twice?

Reply:
Thanks for the suggestion, I agree that's a good idea! I've gone ahead to implement that in https://github.com/Automattic/wp-calypso/pull/68293/commits/a9db374d168efbd01b9f9bf2ce087c86910da5b1.